04/11/23 (22:20:31) | ERROR: [Errno 2] No such file or directory: '/home/sugan/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/sugan/.config/bpytop/themes'
08/07/24 (09:29:42) | ERROR: [Errno 2] No such file or directory: '/home/sugan/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/sbin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/sugan/.config/bpytop/themes'
21/11/24 (13:17:30) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
21/11/24 (13:17:30) | WARNING: No theme named "monokai" found!
21/11/24 (13:17:44) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
21/11/24 (13:17:53) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
14/12/24 (11:21:20) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
14/03/25 (15:53:16) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
14/03/25 (15:53:40) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
14/03/25 (16:50:26) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
14/03/25 (16:52:21) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
24/04/25 (16:34:36) | ERROR: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ~~~~~~~~~~^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sugavanesh/.config/bpytop/themes'
